src/Symfony/Component/Notifier/Bridge/GoogleChat/GoogleChatOptions.php

```diff
@@ -62,7 +62,7 @@ public function toArray(): array
     }
 
     /**
-     * @return $this
+     * @deprecated google chat cards-v1 deprecated, use "cardV2()" instead.
      */
```

Comment:
No need to deprecate here I guess, can't we just change the key in the array and use card() method as public api?

Reply:
I don't think so, in google developer api doc, we can still use card and cardV..
The old cards are not compatible with the v2 so it will break old code.

I will create a new PR to help create cards using a factory instead of juste injecting the all cards array.

Reply:
the expected array structure is not the same